rust/processor/src/models/fungible_asset_models/v2_fungible_metadata.rs

```diff
@@ -96,7 +83,9 @@ impl FungibleAssetMetadataModel {
                     supply_aggregator_table_handle_v1: None,
                     supply_aggregator_table_key_v1: None,
                     token_standard: TokenStandard::V2.to_string(),
-                    is_token_v2: Some(is_token_v2),
+                    is_token_v2: None,
+                    supply_v2,
+                    maximum_v2,
```

Comment:
```
supply_v2: supply_v2,
maximum_v2: maximum_v2,
```
?

Reply:
it's shorthand since they name is same as variable.

Reply:
haha @ying-w nice try.